src/testingUtils.ts

```diff
@@ -71,20 +69,21 @@ export class DummyOrchestrationContext implements IOrchestrationFunctionContext
         // Set this as undefined, let it be initialized by the orchestrator
         this.df = (undefined as unknown) as DurableOrchestrationContext;
     }
-    public doneValue: IOrchestratorState | undefined;
-    public err: string | Error | null | undefined;
+    functionName: string;
+    extraInputs: InvocationContextExtraInputs;
+    extraOutputs: InvocationContextExtraOutputs;
```

Comment:
Would be nice if `DummyOrchestrationContext` extended `InvocationContext` instead of copy/pasting all this stuff

Reply:
Done!